components/content_settings/core/browser/brave_content_settings_pref_provider.cc

```diff
@@ -21,6 +21,7 @@
 #include "brave/components/constants/pref_names.h"
 #include "brave/components/content_settings/core/browser/brave_content_settings_utils.h"
 #include "brave/components/content_settings/core/common/content_settings_util.h"
+#include "brave/components/permissions/google_sign_in/google_sign_in_permission_util.h"
```

Comment:
brave/components/permissions is in components/permissions so it is not allowed here as that would create a dependency cycle. components/permissions should be disallowed in DEPS because it's allowed from components/permissions -> components/content_settings/core